### PR TITLE
Al-amin/35136/Deriv p2p landing page update

### PR DESCRIPTION
**Pull request by alamin-deriv**
Changes:

-  Added Deriv P2P articles on Help center
- Updated Deriv P2P landing page with latest changes
- added universal QR code that works for all app stores
- Added Deriv P2P video tutorial on Deriv P2P landing page

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/p2p/components/_dp2p.js [hunk 1]
```diff
@@ -111,9 +121,25 @@ const DP2P = ({ P2P, reverse, two_title }) => {
...
+                </VideoText>
+                <iframe
+                  width="560"
+                  height="315"
+                  src="https://www.youtube.com/embed/zf9flqE94Ek"
+                  title="YouTube video player"
+                  frameBorder="0"
+                  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                  allowFullScreen>
+                </iframe>
```
Comment (sean-binary) [inline src/pages/p2p/components/_dp2p.js:141]: You're gonna need to extend this and style it for mobile for as low as 320px as well. 🙏

**Review by sean-binary — COMMENTED** [review]
Rest LGTM

File: src/pages/p2p/components/_dp2p.js (hunk 1 above)
Comment (alamin-deriv) [reply src/pages/p2p/components/_dp2p.js:141]: oh yeah sure, let me do that

File: src/pages/p2p/components/_dp2p.js [hunk 2]
```diff
@@ -91,6 +101,15 @@ const Row = styled.div`
...
 `
+
+const StyledIFrame = styled.iframe`
+    width: 560px;
+    height: 315px;
+
+    @media ${device.mobileM} {
+        width: 310px;
+    }
+`
```
Comment (sean-binary) [inline src/pages/p2p/components/_dp2p.js:112]: It's still gonna break from 560px width to mobileM.

Comment (alamin-deriv) [reply src/pages/p2p/components/_dp2p.js:112]: So what is the better width for use here?

Comment (sean-binary) [reply src/pages/p2p/components/_dp2p.js:112]: Better to set the breakpoint for like tablet/tabletS and the width to say like 90% with a max-width for it.

Comment (alamin-deriv) [reply src/pages/p2p/components/_dp2p.js:112]: okay sure, let me work on it

File: src/pages/p2p/components/_dp2p.js [hunk 3]
```diff
@@ -106,8 +106,14 @@ const StyledIFrame = styled.iframe`
...
 
+    @media ${device.tablet} {
+        max-width: 90%;
+    }
+    @media ${device.tabletS} {
+        max-width: 90%;
+    }
     @media ${device.mobileM} {
-        width: 310px;
+        max-width: 90%;
```
Comment (sean-binary) [inline src/pages/p2p/components/_dp2p.js]: I can't fully understand the implementation here. If your intention is to have the max-width of the player to be 560px but when shrunk down to for e.g. tablet to maintain 90% width of the screen. You can do something like this instead:
```
const StyledIFrame = styled.iframe`
    height: 315px;
    width: 100%;
    max-width: 560px;

    @media ${device.tablet} {
        width: 90%;
    }
```

**Review by arome-binary — APPROVED** [review]
LGTM

File: src/pages/p2p/components/_dp2p.js [hunk 4]
```diff
@@ -91,6 +101,21 @@ const Row = styled.div`
...
+    width: 100%;
+    max-width: 560px;
+
+    @media ${device.tablet} {
+        width: 90%;
+    }
+
+    @media ${device.mobileM} {
+        width: 90%;
+    }
```
Comment (sean-binary) [inline src/pages/p2p/components/_dp2p.js:116]:
```suggestion
```

Comment (sean-binary) [reply src/pages/p2p/components/_dp2p.js:116]: Actually in fact I think we can just remove this entirely, just noticed that your container had already paddings included. Last one I promise 🙏

**Review by sean-binary — APPROVED** [review]

**Pull request merged**